concordium-sdk/src/main/java/com/concordium/sdk/responses/BlockInfoV2.java

```diff
@@ -0,0 +1,29 @@
+package com.concordium.sdk.responses;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.transactions.Hash;
+import com.concordium.sdk.types.UInt64;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+/**
+ * Information about the Block. See {@link ClientV2#getBlocks()} & {@link ClientV2#getFinalizedBlocks()} ()}.
+ */
+@RequiredArgsConstructor
+@Builder
+@ToString
+@EqualsAndHashCode
+public class BlockInfoV2 {
```

Comment:
Why can't the existing `BlockInfo` be used?

Reply:
Also there seem to be missing some data?
https://github.com/Concordium/concordium-java-sdk/blob/main/concordium-sdk/src/main/java/com/concordium/sdk/responses/blockinfo/BlockInfo.java